Meshtastic/Extensions/String.swift

```diff
@@ -61,11 +61,11 @@ extension String {
 	}
 
 	func camelCaseToWords() -> String {
-		return unicodeScalars.dropFirst().reduce(String(prefix(1))) {
-			return CharacterSet.uppercaseLetters.contains($1)
-				? $0 + " " + String($1)
-				: $0 + String($1)
-		}
+		return self
+			.replacingOccurrences(of: "([a-z])([A-Z](?=[A-Z])[a-z]*)", with: "$1 $2", options: .regularExpression)
+			.replacingOccurrences(of: "([A-Z])([A-Z][a-z])", with: "$1 $2", options: .regularExpression)
+			.replacingOccurrences(of: "([a-z])([A-Z][a-z])", with: "$1 $2", options: .regularExpression)
+			.replacingOccurrences(of: "([a-z])([A-Z][a-z])", with: "$1 $2", options: .regularExpression)
```

Comment:
Is this last replacingOccurrences() redundant? Looks identical to the prior line.

Reply:
Good catch